src/app/sandbox/deploy/LeftSection.tsx

```diff
@@ -51,67 +52,79 @@ export function LeftSection() {
-            const state = btoa(JSON.stringify(paramsBase64));
+      <Stack gap={3}>
+        <Stack gap={5}>
+          <Flex>
+            <InputGroup
+              border="1px solid transparent"
+              alignItems="center"
+              borderRadius="24px"
+              position="relative"
+            >
+              <Input
+                value={contractName}
+                onChange={(e: any) => setContractName(e.target?.value as string)}
+                placeholder="Name your contract"
+                pr="84px"
+              />
+              <InputRightElement>
+                <IconButton
+                  onClick={() => {
+                    setContractName('');
+                  }}
+                  icon={<Icon as={X} size={4} />}
+                  aria-label={'clear contract name field'}
+                />
+              </InputRightElement>
+            </InputGroup>
+            <Box onClick={() => dispatch(toggleCodeToolbar())}>
+              <Tooltip label="Contract tools">
+                <IconButton
+                  icon={<Icon as={Toolbox} size={4} />}
+                  aria-label={'contract tools'}
+                  size={'40px'}
+                />
+              </Tooltip>
+            </Box>
+          </Flex>
+          <Stack alignItems="center" justifyContent={'center'} gap={2}>
+            <Button
+              onClick={() =>
+                isConnected
+                  ? onDeploy()
+                  : connect({
+                      onFinish: authData => {
+                        dispatch(setUserData({ userData: authData.userSession.loadUserData() }));
+                      },
+                    })
+              }
+              width="100%"
+              variant={'primary'}
+            >
+              {isConnected ? 'Deploy' : 'Connect Stacks Wallet'}
+            </Button>
+          </Stack>
+        </Stack>
+        <Stack gap={2}>
+          <Button
+            variant="secondary"
+            rightIcon={<ArrowSquareOut />}
+            onClick={() => {
+              const paramsBase64 = { name: contractName, sourceCode: codeBody };
+              const state = btoa(JSON.stringify(paramsBase64));
 
-            // 'state' param is used by Auth0 to forward the params after the login portal
-            window.open(`https://platform.hiro.so/projects/import?state=${state}`);
-          }}
-          width="100%"
-        >
-          Save Contract in Hiro Platform
-        </Button>
+              // 'state' param is used by Auth0 to forward the params after the login portal
+              window.open(`https://platform.hiro.so/projects/import?state=${state}`);
+            }}
+            width="100%"
+          >
+            Save Contract in Hiro Platform
+          </Button>
+          <Text fontSize="xs" as="p">
+            The sandbox doesn't support Clarity 3 deployments. To deploy a Clarity 3 contract,
```

Comment:
This is missing the links to docs https://github.com/hirosystems/explorer/issues/1855#issuecomment-2429192330